cmd/bundle.go

```diff
@@ -98,7 +98,7 @@ var bundleDeprovisionCmd = &cobra.Command{
 	Long:  `Deprovision ServiceBundles from a registry adapter`,
 	Args:  cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		preRun("deprovision", args)
+		executedBundle("deprovision", args)
```

Comment:
Think this is a typo, don't think this will build

Reply:
Thanks, oops